### PR TITLE
Feature/redshiftencryption issue changes.

### PR DESCRIPTION
**Pull request by vigneswararaomacharla**
Feature/redshiftencryption issue changes.

### CODE REVIEW

File: deployment/cf-templates/ddb.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
                     }
                 ],
                 "ProvisionedThroughput": {
-                    "ReadCapacityUnits": "10",
+                    "ReadCapacityUnits": "25",
```
Comment (oleksii-dowjones) [inline deployment/cf-templates/ddb.json]: Why did you change it?

File: deployment/cf-templates/identification.json [hunk 2]
```diff
@@ -27,7 +27,8 @@
                         "SourceIdentificationEBSVolumes",
                         "SourceIdentificationEBSSnapshots",
                         "SourceIdentificationRDSSnapshots",
-                        "SourceIdentificationAMIPublicAccess"
```
Comment (oleksii-dowjones) [inline deployment/cf-templates/identification.json]: Why do you have so much changes in this file?

Comment (oleksii-dowjones) [reply deployment/cf-templates/identification.json]: It seems they are unrelated to commit.

File: deployment/cf-templates/reporting-remediation-role.json [hunk 3]
```diff
@@ -206,6 +206,24 @@
...
+                                    "Sid": "RedshiftIssues",
+                                    "Effect": "Allow",
+                                    "Action": [
+                                        "redshift:DescribeClusterSecurityGroups",
+                                        "redshift:DescribeClusterParameterGroups",
+                                        "redshift:DescribeLoggingStatus",
+                                        "redshift:GetClusterCredentials",
+                                        "redshift:DescribeClusters",
+                                        "redshift:EnableLogging",
+                                        "redshift:DisableLogging",
```
Comment (oleksii-dowjones) [inline deployment/cf-templates/reporting-remediation-role.json]: Why do we need to disable logging?

File: deployment/terraform/modules/identification/identification.tf [hunk 4]
```diff
@@ -1,7 +1,8 @@
 resource "aws_cloudformation_stack" "identification" {
-    name = "hammer-identification"
+    name = "hammer-identification-main"
```
Comment (oleksii-dowjones) [inline deployment/terraform/modules/identification/identification.tf]: unrelated changes?

File: hammer/library/aws/redshift.py [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+                                          name=cluster_id,
+                                          tags=tags,
+                                          is_encrypted=is_encrypted,
+                                          is_public=is_public)
+                self.clusters.append(cluster)
+
+        return True
+
+
+class RedshiftLoggingChecker(object):
```
Comment (oleksii-dowjones) [inline hammer/library/aws/redshift.py]: A lot of code repetition in this class.
Can we move common functionality to base class?

File: hammer/reporting-remediation/remediation/clean_redshift_cluster_unencrypted.py [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+                if in_whitelist:
+                    logging.debug(f"Skipping {cluster_id} (in whitelist)")
+                    # Adding label with "whitelisted" to jira ticket.
+                    jira.add_label(
+                        ticket_id=issue.jira_details.ticket,
+                        labels=IssueStatus.Whitelisted
+                    )
+                    continue
+                if not in_fixlist:
```
Comment (oleksii-dowjones) [inline hammer/reporting-remediation/remediation/clean_redshift_cluster_unencrypted.py]: It's always evaluated to `False`. Why do we need it?

File: hammer/reporting-remediation/reporting/create_redshift_unencrypted_cluster_issue_tickets.py [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+                            msg=f"{comment}"
+                                f"{' (' + jira.ticket_url(issue.jira_details.ticket) + ')' if issue.jira_details.ticket else ''}",
+                            owner=owner,
+                            account_id=account_id,
+                            bu=bu, product=product,
+                        )
+                        IssueOperations.set_status_closed(ddb_table, issue)
+                    # issue.status != IssueStatus.Closed (should be IssueStatus.Open)
+                    elif issue.timestamps.updated > issue.timestamps.reported:
+                        logging.error(f"TODO: update jira ticket with new data: {table_name}, {account_id}, {cluster_id}")
```
Comment (oleksii-dowjones) [inline hammer/reporting-remediation/reporting/create_redshift_unencrypted_cluster_issue_tickets.py]: Why do we need this error in logs?

File: deployment/terraform/modules/identification/identification.tf (hunk 4 above)
Comment (vigneswararaomacharla) [reply deployment/terraform/modules/identification/identification.tf]: While resolving conflicts, we have taken these changes to this branch from dev..

File: deployment/cf-templates/identification.json (hunk 2 above)
Comment (vigneswararaomacharla) [reply deployment/cf-templates/identification.json]: While resolving conflicts, we have taken these changes to this branch from dev..

File: deployment/cf-templates/ddb.json (hunk 1 above)
Comment (vigneswararaomacharla) [reply deployment/cf-templates/ddb.json]: While resolving conflicts, we have taken these changes to this branch from dev.

**Review by oleksii-dowjones — APPROVED** [review]
